documentation/importing-dependencies/tutorial-include-css.asciidoc

```diff
@@ -37,48 +41,44 @@ annotation.
 }
 ----
 
-You can place style sheets and other static resources in any folder inside your WAR file except for `/VAADIN` which is reserved for framework internal use.
-`VaadinServlet` handles static resource requests if you have mapped it to `/*` .
-Otherwise, the servlet container will take care of static resource requests.
+== Storing CSS Files
+
+You can place CSS style sheets and other static resource files in any folder in your WAR file, except `/VAADIN`, which is reserved for internal framework use.
+
+`VaadinServlet` handles static resource requests, if you have mapped it to `/*`.
+If not, the servlet container takes care of static resource requests.
+
+If you use relative URLs, it is irrelevant whether your application is deployed in the root context (for example  `\http://mysite.com/`) or in a sub context (for example `\http://mysite.com/myapp/`). Relative URLs are resolved using the page base URI, which is always set to match the servlet URL.
 
-By using relative URLs you are not dependent on whether the application is deployed in the root context (e.g.  `http://mysite.com/`) or in a sub context (e.g. `http://mysite.com/myapp/`).
-Relative URLs are resolved using the page base URI, which is always set to match the servlet URL.
+=== Using a Servlet Path
 
-[TIP]
-====
-If you are using a servlet path for the servlet, e.g. `http://mysite.com/myapp/myservlet/` then you will need to take the servlet path into account when including resources.
-This is needed because the base URI is `http://mysite.com/myapp/myservlet/` but static resources are deployed in `http://mysite.com/myapp/`.
+If you use a servlet path for the servlet, for example `\http://mysite.com/myapp/myservlet/`, you need to take the servlet path into account when including resources. This is because the base URI is `\http://mysite.com/myapp/myservlet/`, but static resources are deployed at `\http://mysite.com/myapp/`.
 
-You can use the special protocol `context://` with e.g. `Page.addStyleSheet` to ensure a URL relative to the context path but this is only supported when including resources.
+You can use the `context://` protocol, with the `Page.addStyleSheet` method, for example. This ensures the URL is relative to the context path, but is only supported when including resources.
 
-When you configure an element, e.g setting the `src` attribute for an `<img>`, you cannot use the `context://` protocol. Your options are then:
+When you configure an element, for example setting the `src` attribute for an `<img>`, you cannot use the `context://` protocol. Your options are:
 
-* Cancel out the servlet path, e.g. `../images/logo.png`.
-* Use an absolute URL, e.g. `/myapp/images/logo.png`
-* Deploy your static resources in a directory matching the servlet path, e.g. `/myservlet/`.
-====
+* Cancel out the servlet path, for example `../images/logo.png`.
```

Comment:
Cancel out sounds quite odd to me. Maybe this would be better?
```suggestion
* Take the servlet path into account with your relative URL, for example `../images/logo.png`.
```